packages/amplify-cli-core/API.md

```diff
@@ -529,7 +529,7 @@ export type CommandLineInput = {
     plugin?: string;
     command: string;
     subCommands?: string[];
-    options?: CLIGlobalFlags & Record<string, any>;
+    options?: CLIGlobalFlags & Record<string, $TSAny>;
```

Comment:
Why make this change from `any` to `$TSAny`?

Reply:
TLDR; wanted to take care of the warning. Posted in brainstorming about wanting to discuss proper path forward.